io.github.endless_sky.endless_sky.json

```
@@ -0,0 +1,71 @@
+{
+  "id": "io.github.endless_sky.endless_sky",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "endless-sky",
+  "rename-desktop-file": "endless-sky.desktop",
+  "rename-icon": "endless-sky",
+  "finish-args": [
+    "--socket=pulseaudio",
+    "--socket=wayland",
+    "--socket=x11",
+    "--device=dri"
+  ],
+  "modules": [
+    "shared-modules/glu/glu-9.json",
+    "shared-modules/glew/glew.json",
+    "shared-modules/libmad/libmad.json",
+    {
+      "name": "scons",
+      "cleanup": [ "*" ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "python3 setup.py install --prefix=/app"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://downloads.sourceforge.net/project/scons/scons/3.1.1/scons-3.1.1.tar.gz",
+          "sha256": "4cea417fdd7499a36f407923d03b4b7000b0f9e8fd7b31b316b9ce7eba9143a5"
+        }
+      ]
+    },
+    {
+      "name": "endless-sky",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ln -s /bin/python3 /app/bin/python",
+        "CPPPATH=/app/include scons PREFIX=/app install -j$(nproc)",
```

```suggestion
        "CPPPATH=/app/include python3 /app/bin/scons PREFIX=/app install -j $FLATPAK_BUILDER_N_JOBS",
```
That's the change I wanted to suggest

---

Thank you!